.github/workflows/aws-e2e.yaml

```diff
@@ -97,8 +97,8 @@ jobs:
     if: |
       github.event_name == 'pull_request' &&
       (
-        (github.event.action == 'labeled' &&  github.event.label.name == 'runs-e2e-tests') ||
-        (github.event.action == 'synchronize' && contains(github.event.pull_request.labels.*.name, 'runs-e2e-tests'))
+        (github.event.action == 'labeled' &&  (github.event.label.name == 'runs-e2e-tests' || github.event.label.name == 'runs-aws-tests')) ||
```

Comment:
`contains` can be used to match string arrays. https://docs.github.com/en/actions/learn-github-actions/expressions#example-matching-an-array-of-strings
```
contains(fromJSON('["runs-e2e-tests", "runs-aws-tests"]'), github.event.label.name)
```
However I am planning to change this part in future PR anyways. lets go with it what we have now.